includes/admin/tools/import/class-give-import-donations.php

```diff
@@ -724,6 +724,7 @@ public function render_media_csv() {
 			if ( empty( $csv_id ) || ! $this->is_valid_csv( $csv_id, $csv ) ) {
 				$csv_id = $csv = '';
 			}
+			$per_page = absint( isset( $_POST['per_page'] ) ? give_clean( $_POST['per_page'] ) : self::$per_page );
```

Comment:
@raftaar1191  can you explain `absint( isset( $_POST['per_page'] )`

Reply:
Update the code 

Thanks @ravinderk for pointing this out to me.